go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/ffledgling/pdns-go v0.0.0-20180219074714-524e7daccd99
 	github.com/go-gandi/go-gandi v0.7.0
 	github.com/go-logr/logr v1.4.2
+	github.com/go-task/slim-sprig/v3 v3.0.0
```

Comment:
the only concern is that the libary had last release 2+ years ago in 2023. Could you justify/convince the library maintainer to 

- refresh a release tag 
- refresh go.mod with go.sum
- would be nice to sync `go` version as well, but not critical

have you checked this library for known vulnerabilities?

```
trivy repo --scanners vuln .
```

or with snyk open source checkers

Reply:
This actually quite tricky. For example github.com/Masterminds/sprig is heavier, but it has 4k stars and almost ~100 contributors.... So worth to share a comparison on why this library vs upstream one

Reply:
It doesn't make a difference for my use case. I only need the `replace` function. Context [here](https://github.com/kubernetes-sigs/external-dns/pull/3949#issuecomment-1736469058). Would you prefer importing github.com/Masterminds/sprig instead, or even creating a similar function here and avoiding the import altogether?

Reply:
It doesn't make a difference for my use case. I only need the `replace` function. Context [here](https://github.com/kubernetes-sigs/external-dns/pull/3949#issuecomment-1736469058). Would you prefer importing github.com/Masterminds/sprig instead, or even creating a similar function here and avoiding the import altogether?

Reply:
I have no opinion. If you refactor logic as per review, and cover functions with tests, we will decide on a library at the very end. Both libraries have they advantages. Or as you mention, we may not need any libraries at this time, as only few functions required.

Reply:
If you only need one function - so no import required. Better to creat own function. It's more about compatibility, we should have tests for all the functions provided.